code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm

```diff
@@ -538,28 +548,61 @@
 	..()
 	return
 
+/datum/reagent/medicine/sleeptoxin
```

Comment:
Nit picking but wouldn't this be better as a toxin with toxpower being 0?
Chloral Hydrate does the same thing.